gemfiles/rails_4.gemfile

```diff
@@ -7,6 +7,7 @@ gem "bundler", "< 2"
 gem "climate_control", "~> 0.2"
 gem "pry"
 gem "pry-byebug"
+gem "mysql2"
```

Comment:
Do we need a hard dependency on `mysql2`?

Reply:
Only for testing. This is because of the explicit checks that we're doing and testing around only supporting `charset` and `collation` initially for `MySQL` connections.

Reply:
Ah, good point--this it the GemfileI(s), not the gemspec. So no big deal.